src/ops/ops.go

```diff
@@ -648,11 +648,12 @@ func (o *ops) CreateManifests(kubeconfig string, content []byte) error {
 	return nil
 }
 
-// DryRebootHappened checks if a reboot happened according to a particular reboot marker path
-// The dry run installer creates this file on "Reboot" (instead of actually rebooting)
-// We use this function to check whether the given node in the cluster have already rebooted
-func (o *ops) DryRebootHappened(markerPath string) bool {
```

Comment:
IMO lets leave this one as it helps to understand that we are looking for dry reboot logic, it can though call for FileExists